project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
error popup visibility is broken 
<img width="471" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/9d82d450-11ee-48b9-88a7-5c8a9a03243a">